### ISSUE TITLE
(feature) Allow partial update of releases in output file.

### ISSUE THREAD

**Issue by hastebrot** [entera/gradle-relnotes#5]
**TL;DR:** The plan is to have the Gradle tasks `createReleaseNotes` and `updateReleaseNotes`. The former a complete creation for all releases, the latter a partial update for the latest releases.
## The Problem

We generate release notes and put them into output files like `CHANGES.md`. These files contain releases, which are the basic building blocks and listed in reverse chronological order.

There are two parts that can be modified for partial updates:
- **output file**: use existing output file and add new releases to the top.
- **network traffic**: only fetch data related to new releases.
